client/api_group.go

```diff
@@ -23,72 +23,76 @@ import (
-	// DeleteGroupPolicy  delete group policy of the principal, the sender need to be the owner of the group
-	// principalAddr indicates the HEX-encoded string of the principal address
 	DeleteGroupPolicy(ctx context.Context, groupName string, principalAddr string, opt types.DeletePolicyOption) (string, error)
-	// GetBucketPolicyOfGroup get the bucket policy info of the group specified by group id
-	// it queries a bucket policy that grants permission to a group
 	GetBucketPolicyOfGroup(ctx context.Context, bucketName string, groupId uint64) (*permTypes.Policy, error)
-	// GetObjectPolicyOfGroup get the object policy info of the group specified by group id
-	// it queries an object policy that grants permission to a group
 	GetObjectPolicyOfGroup(ctx context.Context, bucketName, objectName string, groupId uint64) (*permTypes.Policy, error)
-	// GetGroupPolicy get the group policy info of the user specified by principalAddr
 	GetGroupPolicy(ctx context.Context, groupName string, principalAddr string) (*permTypes.Policy, error)
-	// ListGroup get the group list by name and prefix.
-	// prefix is the start of the search pattern. The system will only return groups that start with this prefix.
-	// name is the ending of the search pattern.
-	// it providers fuzzy searches by inputting a specific name and prefix
 	ListGroup(ctx context.Context, name, prefix string, opts types.ListGroupsOptions) (types.ListGroupsResult, error)
-	// RenewGroupMember renew a list of group members and their expiration time
 	RenewGroupMember(ctx context.Context, groupOwnerAddr, groupName string, memberAddresses []string, opts types.RenewGroupMemberOption) (string, error)
-	// ListGroupMembers returns a list of members contained within the group specified by the group id, including those for which the user's expiration time has already elapsed
 	ListGroupMembers(ctx context.Context, groupID int64, opts types.GroupMembersPaginationOptions) (*types.GroupMembersResult, error)
-	// ListGroupsByAccount returns a list of all groups that the user has joined, including those for which the user's expiration time has already elapsed
-	// By default, the user is the sender. Other users can be set using the option
 	ListGroupsByAccount(ctx context.Context, opts types.GroupsPaginationOptions) (*types.GroupsResult, error)
-	// ListGroupsByOwner returns a list of groups owned by the specified user, including those for which the user's expiration time has already elapsed
-	// By default, the user is the sender. Other users can be set using the option
 	ListGroupsByOwner(ctx context.Context, opts types.GroupsOwnerPaginationOptions) (*types.GroupsResult, error)
 }
 
-// CreateGroup create a new group on greenfield chain, the group members can be initialized or not
+// CreateGroup - Create a new group without group members on Greenfield blockchain, and group members can be added by UpdateGroupMember transaction.
+//
```

Comment:
Add more details for this API, and introduce the group concept here, because it is quite a different concept from other chains.

Reply:
added